README.md

```diff
@@ -1,180 +1,113 @@
-
-In the two above cases, you will need to setup a jupyter kernel that points to your `<env_name>` environment, to be able to find and execute successfully the qiskit-metal package.
-
-Check for the instructions to install a new kernel in the [FAQ](https://qiskit.org/documentation/metal/faq.html).
-
-#### Subsequent updates of the conda environment
-
-Package dependencies will evolve over time and could at some point require a new version of a library.
-For example, we can anticipate updating `pyEPR-quantum` to enable Ansys interactions previously unsupported.
-To update your local install, simply execute the metal package install command
-```sh
-conda env update -n <env_name_exist> environment.yml
-``` 
-Alternatively, you can remove your conda environment by executing the commands below and later re-create a new environment following the original install instructions in section 1.
-```sh
-conda env list
-conda env remove -n <env_name_exist>
-``` 
-Notice that using the `conda env update` command might introduce inconsistencies in your virtual environment, and render it unusable. This occurs in general when using conda install commands after any number of pip install commands.
-
-### Setup without conda: in a virtual environment (alternative setup)
-
-#### Prerequisites
-The package dependency `gdspy`, needs C++ compilation to successfully install in a base or virtualenv. Make sure the right compiler is installed on your machine if you encounter errors during the installation process described above.
-**Windows** you can install the `Visual C++ x.0` using the [C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/). Be sure to select the most current versions of `MSVC` and `Windows SDK`, as suggested in [this wiki](https://wiki.python.org/moin/WindowsCompilers) referenced by the gdspy documentation.
-**Linux** on Ubuntu or other Debian based systems, execute the following command `sudo apt-get install gcc g++`. Linux users might encounter other python-related errors during creation of the virtualenv or during qiskit-metal installation. You might be able to deal with those by executing `sudo apt-get install python3-dev python3-venv`. Be sure to customize the `python3` string if you are trying to use a custom installation of python.
-
-#### Install
-To create and populate the Python virtual environment, execute these commands in the top-level of the repository:
-```sh
-python -m venv <virtual_env_path/name>
-source <virtual_env_path/name>/bin/activate
-python -m pip install -U pip
-python -m pip install -e .
+```python
+>>> from qiskit_metal import designs, draw, MetalGUI, Dict, open_docs
+>>> design = designs.DesignPlanar()
+>>> design.overwrite_enabled = True
+>>> design.chips.main
+>>> design.chips.main.size.size_x = '11mm'
+>>> design.chips.main.size.size_y = '9mm'
+>>> gui = MetalGUI(design)
+```
+#### Launch the Qiskit Metal GUI to interactively view, edit, and simulate a QDesign:
```

Comment:
These are great Xmas Mark I would suggest adding some images in here as well to make it more visual